src/playground/addons/storybook-addon-system-switcher/systemSwitcher.jsx

```diff
@@ -3,10 +3,9 @@ import { IconButton, Separator } from '@storybook/components';
 
 const Switcher = () => {
   const queryString = window.location.search;
-  let system = window.location.pathname.split('/').filter((p) => p)[1];
-  if (!system) {
-    system = process.env.STORYBOOK_SYSTEM ? 'eu' : 'ec';
-  }
+  const system =
```

Comment:
Noticing a change here in the switcher. We need to have https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1838 soon in order to avoid difference in system extraction logic.

Here's the way I did it in the code addon, taking env variable first and first system-specific match from path as a second.

```javascript
const system = (
  window.location.pathname
    .split('/')
    .find((part) => part === 'ec' || part === 'eu') ||
  process.env.STORYBOOK_SYSTEM
).toLowerCase();
```